### PR TITLE
appendix: review/add environment variables

### LINKED ISSUE #181: 4.0: Add additional environment variables to appendix
Review the CE 4.0.0 changelog for flag additions / changes, and add / update any relevant environment variables in appendix.rst

### PR DESCRIPTION
**Pull request by dtrudg**
Add new environment variables for 4.0

Review the listed environment variables vs those present in the CLI code. Add / rename / edit as appropriate.

Fixes #181

### CODE REVIEW

**Review by wobito — DISMISSED** [review]

File: appendix.rst [hunk 1]
```diff
@@ -224,6 +253,22 @@ below with their respective functionality.
 #. **SINGULARITY_LIBRARY**: Specifies the library to pull from. Default
    is set to our Cloud Library.
 
+#. **SINGULARITY_LOCAL_VERIFY**: Set to true to only use the local keyring when
+   verifying PGP signe SIF images. Disables retrieval of public keys from
```
Comment (preminger) [inline appendix.rst]:
```suggestion
   verifying PGP signed SIF images. Disables retrieval of public keys from
```

File: appendix.rst [hunk 2]
```diff
@@ -303,31 +372,44 @@ below with their respective functionality.
...
+   Note that ``library://`` pulls ignore the platform variant.
 
 ``R``
 =====
 
-#. **SINGULARITY_REMOTE**: To build an image remotely. (Does not require
-   root) Default is set to false.
-#. **SINGULARITY_ROOTFS**: To reference the system file location.
+#. **SINGULARITY_REMOTE**: Set to true to build an image remotely. (Does not
+   require root) Default is set to false.
```
Comment (preminger) [inline appendix.rst]:
```suggestion
   require root.) Default is set to false.
```

File: appendix.rst [hunk 3]
```diff
@@ -303,31 +372,44 @@ below with their respective functionality.
...
+
+#. **SINGULARITY_ROOTFS**: Set to true to reference the system file location.
```
Comment (preminger) [inline appendix.rst]: Personally, I found this description unclear...

File: appendix.rst [hunk 4]
```diff
@@ -344,14 +426,21 @@ below with their respective functionality.
    mount SIF images with ``squashfuse`` in unprivileged user namespace
    workflows.
 
-#. **SINGULARITY_SIGNAL**: Specifies a signal sent to the instance.
+#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance.
```
Comment (preminger) [inline appendix.rst]:
```suggestion
#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance when stopping it.
```

**Review by preminger — APPROVED** [review]
LGTM, but see comments/suggestions.

File: appendix.rst (hunk 4 above)
Comment (dtrudg) [reply appendix.rst]: Hmm - this is an interesting one, as you could send a non-kill/term signal to the instance which doesn't stop it.... but the command is called stop.

Comment (preminger) [reply appendix.rst]: Not that different from unix `kill`, when you think about it... (can also be used to send signals that don't "kill" the recipient process).

Comment (dtrudg) [reply appendix.rst]: That's my point really. It is unix kill. This means that `singularity instance stop` with a specified signal isn't always stopping the container.

Comment (dtrudg) [reply appendix.rst]:
```
#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance with
   ``singularity instance stop``.
```

**Pull request merged**